src/lib/plugins/moderation.js

```diff
@@ -119,7 +119,9 @@ module.exports.server = function (serv, settings) {
     },
     action (params) {
       params = params.split(' ')
-      const player = serv.getPlayer(params[0])
+      const [inputUsername] = params.split(' ')
+      // get player, by non-case-sensitive username
+      const player = serv.players.find(player => player.username.toLowerCase() === inputUsername.toLowerCase())
```

Comment:
It's probably better to modify serv.getPlayer instead since this check would be useful for other commands as well.

Reply:
I copied the code from /op command 
So i guess its ok